server/theme.lua

```diff
@@ -0,0 +1,40 @@
+local themes = Config.theme.themes
+-- TODO: Do we need to store this path somewhere central so it can be used in other resources?
+local filePath = GetResourcePath('qb-core') .. "/html/css/themes.css"

```

Comment:
As multiple resources will need to access this file path, it's probably worth storing this somewhere central.

Reply:
A function for `getThemePath` might be a nice addition